tests/e2e/tests/product-collection/compatibility-layer-block_theme.spec.ts

```diff
@@ -0,0 +1,150 @@
+/**
+ * External dependencies
+ */
+import { test as base, expect } from '@woocommerce/e2e-playwright-utils';
+
+/**
+ * Internal dependencies
+ */
+import ProductCollectionPage from './product-collection.page';
+import {
+	installPluginFromPHPFile,
+	uninstallPluginFromPHPFile,
+} from '../../mocks/custom-plugins';
```

Comment:
We are trying to importing the utils avoiding the relative import:

```suggestion
} from 'from '@woocommerce/e2e-mocks/custom-plugins';
```